### PR TITLE
Service Worker docs: Update cache.put arguments in example

### PR DESCRIPTION
**Pull request by ObieMunoz**
Cache.put takes two arguments but is only supplied one in the example.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: documentation/docs/30-advanced/40-service-workers.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ self.addEventListener('fetch', (event) => {
 			const response = await fetch(event.request);
 
 			if (response.status === 200) {
-				cache.put(response.clone());
+				cache.put(url.pathname, response.clone());
```
Comment (Rich-Harris) [inline documentation/docs/30-advanced/40-service-workers.md:75]:
```suggestion
				cache.put(event.request, response.clone());
```

**Review by Rich-Harris — APPROVED** [review]

**Comment by ObieMunoz** [comment]
@Rich-Harris ah thanks for correcting it to event.request from pathname :)

**Comment by Rich-Harris** [comment]
whoops! good catch, thanks. changed it to `event.request`

**Comment by Rich-Harris** [comment]
gah, github UI, always one step behind the last comment

**Pull request merged**